pkg/mcs/resourcemanager/server/config.go

```diff
@@ -71,8 +68,6 @@ type Config struct {
 	ListenAddr          string `toml:"listen-addr" json:"listen-addr"`
 	AdvertiseListenAddr string `toml:"advertise-listen-addr" json:"advertise-listen-addr"`
 	Name                string `toml:"name" json:"name"`
-	DataDir             string `toml:"data-dir" json:"data-dir"` // TODO: remove this after refactoring
```

Comment:
Will removing these configurations cause compatibility issues?

Reply:
AFAIK, the deploy tool doesn't use the data-dir in the start script.